src/content/docs/dropins/cart/slots.mdx

```diff
@@ -98,166 +135,169 @@ export interface Item {
+      wrapper.appendChild(discount);
+    });
+  });
+},
 ```
 
-### ProductAttributes
+### ProductAttributes slot
 
-The `ProductAttributes` slot allows you to add content to the `CartSummaryList` container.
+The `ProductAttributes` slot allows you to add custom content to the `CartSummaryList` container.
 
 The following example demonstrates how to add the Fashion Material and Fashion Style product attributes to each product in the cart.
 
 ```javascript
-return provider.render(CartSummaryList, {
-  enableRemoveItem: true,
-  enableUpdateItemQuantity: true,
-  routeProduct: (item) => {
-    return `${item.url.categories.join('/')}/${item.url.urlKey}`;
-  },
-  routeEmptyCartCTA: () => '#empty-cart',
-  slots: {
-    ProductAttributes: (ctx) => {
-      // Prepend Product Attributes
-      const ProductAttributes = ctx.item?.productAttributes;
-
-      ProductAttributes?.forEach((attr) => {
-        if (attr.code === 'Fashion Material' || attr.code === 'Fashion Style') {
-          if (attr.selected_options) {
-            const selectedOptions = attr.selected_options
+  return provider.render(CartSummaryList, {
+    enableRemoveItem: true,
+    enableUpdateItemQuantity: true,
+    routeProduct: (item) => {
+      return `${item.url.categories.join('/')}/${item.url.urlKey}`;
+    },
+    routeEmptyCartCTA: () => '#empty-cart',
+    slots: {
+      ProductAttributes: (ctx) => {
+        // Prepend Product Attributes
+        const ProductAttributes = ctx.item?.productAttributes;
+
+        ProductAttributes?.forEach((attr) => {
+          if(attr.code === "Fashion Material" || attr.code === "Fashion Style") {
+            if(attr.selected_options) {
+              const selectedOptions = attr.selected_options
               .filter((option) => option.label.trim() !== '')
               .map((option) => option.label)
               .join(', ');
-
-            if (selectedOptions) {
+              
+              if(selectedOptions) {
+                const productAttribute = document.createElement('div');
+                productAttribute.innerText = `${attr.code}: ${selectedOptions}`;
+                ctx.appendChild(productAttribute);
+              }
+            } else if (attr.value) {
               const productAttribute = document.createElement('div');
-              productAttribute.innerText = `${attr.code}: ${selectedOptions}`;
+              productAttribute.innerText = `${attr.code}: ${attr.value}`;
               ctx.appendChild(productAttribute);
             }
-          } else if (attr.value) {
-            const productAttribute = document.createElement('div');
-            productAttribute.innerText = `${attr.code}: ${attr.value}`;
-            ctx.appendChild(productAttribute);
           }
-        }
-      });
+        })
+      },
+    },
+  })(productList);
+```
+
+### CartSummaryFooter slot
+
+The `CartSummaryFooter` slot allows you to add content to the bottom of the `CartSummaryList` container.
```

Comment:
I found myself wondering what the difference between this and the footer slot on line 210. Maybe it's obvious to developers looking at the code samples, but it isn't to me.

Reply:
The piece needed is in the Footer code example. It was missing the wrapper `Footer: () => {}` within `slots: {}`